platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -290,7 +290,7 @@ public void addTrustedCertificate(String alias, String keyPassword, String store
     private boolean validKeystoreAlias(String alias, String keystorePassword, String keystoreData,
             String keystoreFileName) throws KeystoreEditorException {
         boolean valid = false;
-        try (InputStream inputStream = new ByteArrayInputStream(Base64.decode(keystoreData))) {
+        try (InputStream inputStream = new ByteArrayInputStream(Base64.getDecoder().decode(keystoreData))) {
```

Comment:
Did you manually test to verify that the Java Base64 decoder can decode the JavaScript Base64 data from the web app?